src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,460 @@
+import * as React from 'react'
+import * as PropTypes from 'prop-types'
+
+import { Extendable } from '../../../types/utils'
+import { ComponentSlotStyle } from '../../themes/types'
+import Downshift, { DownshiftState, StateChangeOptions, A11yStatusMessageOptions } from 'downshift'
+import Label from '../Label/Label'
+import { AutoControlledComponent, customPropTypes } from '../../lib'
+import Input from '../Input/Input'
+import keyboardKey from 'keyboard-key'
+import List from '../List/List'
+import Text from '../Text/Text'
+import Image from '../Image/Image'
+import ListItem from '../List/ListItem'
+import _ from 'lodash'
+import Icon from '../Icon/Icon'
+
+export interface DropdownProps {
+  className?: string
+  fluid?: boolean
+  getA11yStatusMessage?: (options: A11yStatusMessageOptions<DropdownListItem>) => string
+  getA11ySelectedMessage?: (item: DropdownListItem) => string
+  getA11yRemovedMessage?: (item: DropdownListItem) => string
+  getA11yRemoveItemMessage?: (item: DropdownListItem) => string
```

Comment:
We should aim to keep all accessibility staff in behaviors. Already discussed with @silviuavram, as this PR is big enough, we have to consider to make it in the future PR.